packages/devextreme-scss/scss/widgets/base/chat/layout/chat-messagegroup/_mixins.scss

```diff
@@ -0,0 +1,30 @@
+@mixin chat-messagegroup(
+  $gap,
+  $padding-top,
+  $start-template,
+  $information-font-size,
+  $information-color,
+  $author-name-gap
+) {
+  .dx-chat-message-group {
+    row-gap: $gap;
+    
+    &:not(&:first-child) {
+      padding-top: $padding-top;
+    }
+  }
+  
+  .dx-chat-message-group-alignment-start {
+    grid-template-columns: $start-template;
+  }
+  
+  .dx-chat-message-time,
```

Comment:
Scenario where should be ellipses are not considered
<img width="461" alt="image" src="https://github.com/user-attachments/assets/f8ea5aba-acfe-49d7-b9ad-86c090d9fe5c">

Reply:
[6bb9fb60543e9af789d9496edf5868b907ba32a8]